src/schema/users.ts

```diff
@@ -751,6 +751,38 @@ const timestampAtTimezone = `"timestamp"::timestamptz ${userTimezone}`;
 
 const MAX_README_LENGTH = 10_000;
 
+export const cachePrefillMarketingCta = async (
```

Comment:
Maybe move it to common/marketingCta just to avoid routes importing schema files, trying to get the imports in order so we don't have so many circular dependencies.

Reply:
I'll see if I find a good spot for it. I think I tried to move the `getMarketingCta` at one point, which really messed up tests.

Reply:
Yeah, I remember that one, let's see 🙏

Reply:
I was able to move it